astacus/client.py

```diff
@@ -51,7 +51,7 @@ def _run_op(op, args, *, json=None, data=None) -> bool:
 
 
 def _run_backup(args) -> bool:
-    return _run_op("backup", args)
+    return _run_op("deltabackup" if args.delta else "backup", args)
```

Comment:
They are different enough that I think it would be better to have a separate command for it instead of an option. (For clarity, to keep the parallel with the coordinator API, and because if we add options to only backup or only delta backups, the argparse and help will be simpler and avoid "this option only applies if you also enable this other option").